### ISSUE TITLE
CytoTRACE tutorial outdated

### ISSUE THREAD

**Issue by WeilerP** [theislab/cellrank#815]
## Description

If I am not mistaken, the API for the CytoTRACE kernel changed with #791. Previously, the all statistics were calculated when instantiating the kernel, _i.e._,

```python
ctk = CytoTRACEKernel(adata)
```

To achieve the same behaviour, the following code needs to be used

```python
ctk = CytoTRACEKernel(adata).compute_cytotrace()
```

The tutorial has not yet been updated accordingly.

**Comment by WeilerP** [comment]
@michalk8, is there a similar change in other kernels?

**Comment by michalk8** [comment]
> @michalk8, is there a similar change in other kernels?

As far as I remember, no, only this has been split from the ``__init__``.

**Comment by Marius1311** [comment]
Thanks for spotting this @WeilerP! Would you mind updating the tutorial accordingly?

**Comment by WeilerP** [comment]
@michalk8, is this even possible? As far as I can tell, the notebooks at stable and latest are the same, right? So changing the notebook would change it for the stable docs as well even though there, the new API is not yet used. Or am I missing something?

**Comment by michalk8** [comment]
> @michalk8, is this even possible? As far as I can tell, the notebooks at stable and latest are the same, right? So changing the notebook would change it for the stable docs as well even though there, the new API is not yet used. Or am I missing something?

You're right, the notebooks aren't versioned into `master`/`dev`. Think we should sooner or later make `master` the development branch again.
